<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QPushButton" name="btn_cliente">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>Cliente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Scegli l'operazione:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_magazzino">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string>Magazzino</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_acquisto">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>85</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string>Acquisto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_backup">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>151</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_statistche">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>365</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistiche</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
